include/platform/radio.h

```diff
@@ -164,7 +189,7 @@ ThreadError otPlatRadioSetShortAddress(uint16_t aShortAddress);
 /**
  * Enable the radio.
  *
- * @retval ::kThreadError_None  Successfully transitioned to Idle.
+ * @retval ::kThreadError_None  Successfully transitioned to Sleep.
```

Comment:
Doesn't "None" returned by otPlatRadioEnable mean successful transition to "Receive"?